src/components/pages/dashboard/dashboard.js

```diff
@@ -215,14 +224,47 @@ export class Dashboard extends Component {
       );
 
       // Start polling all panels
-      this.dashboardRefresh$.next('r');
+      if (this.props.deviceLastUpdated) {
```

Comment:
nit: space after if or [not](https://github.com/Azure/pcs-remote-monitoring-webui/pull/925/files#diff-cf8bd6b6f21705cbd68f1dc5ad34bba3R242) and [here](https://github.com/Azure/pcs-remote-monitoring-webui/pull/925/files#diff-5b2eda635d9ab9d1cd08e5a65c2affb9R136) also ?

Reply:
Should be a space in the linked to locations.